rest/cluster-pending-devices-get.rst

```diff
@@ -0,0 +1,17 @@
+GET /rest/cluster/pending/devices
+=================================
+
+.. versionadded:: 1.FIXME
```

Comment:
Probably 1.13.0 :)

Reply:
I wasn't sure if it would be v1.12.2.  I assume there is something already baking to justify v1.13.

Reply:
This. It's a config and api change.

Reply:
Well explained. :-)